nautobot-app-chatops/{{ cookiecutter.project_slug }}/development/nautobot_config.py

```diff
@@ -95,11 +95,11 @@
         "disable_existing_loggers": False,
         "formatters": {
             "normal": {
-                "format": "%(asctime)s.%(msecs)03d %(levelname)-7s %(name)s :\n  %(message)s",
+                "format": "%(asctime)s.%(msecs)03d %(levelname)-7s %(name)s :  %(message)s",
```

Comment:
Consider replacing the two spaces after the colon with a single space in each case, since it's not serving the purpose of indentation any more?

Reply:
Good catch, done.